docroot/modules/custom/uiowa_core/src/Entity/Card.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\uiowa_core\Entity;
+
+use Drupal\block_content\Entity\BlockContent;
+use Drupal\Core\Render\Element;
+
+/**
+ * A bundle entity class for card block content.
+ */
+class Card extends BlockContent implements RendersAsCardInterface {
+
+  use RendersAsCardTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildCard(array &$build) {
+    $this->buildCardStyles($build);
+
+    // Add fields to card.
+    $this->mapFieldsToCardBuild($build, [
+      '#media' => 'field_uiowa_card_image',
+      '#subtitle' => 'field_uiowa_card_author',
+      '#content' => 'field_uiowa_card_excerpt',
+    ]);
+
+    // @todo Capture the parts of the URL. This isn't working with
```

Comment:
Is this still an issue that should be handled in this PR?

Reply:
going to confirm this is no longer an issue

Reply:
checks out https://github.com/uiowa/uiowa/pull/6054/commits/d6f5a0adebd98ebc183c44004cbf4e18ab51295e